Classes/Command/NamespaceCommandController.php

```diff
@@ -86,11 +84,10 @@ protected function getDeprecatedFileTemplate()
     {
         return <<<'EOD'
 <?php
-namespace {{oldNamespace}}
+namespace {{oldNamespace}};
 
 /*
- * This class namespace was deprecated and moved to {{newNamespace}},
- * find the replacement here: {{newPhpFile}}
+ * This class namespace was deprecated and moved to the new PSR-4 namespace {{newNamespace}}
```

Comment:
How can a namespace be PSR-4? That is for class file locations, not the namespace as such…